specification/cognitiveservices/Language.AnalyzeText/main.tsp

```diff
@@ -0,0 +1,108 @@
+@service({
+  title: "Microsoft Cognitive Language Service - Text Analysis",
+})
+@server(
+  "{Endpoint}/language",
+  "Single server endpoint",
+  {
+    @doc("Supported Cognitive Services endpoint (e.g., https://<resource-name>.api.cognitiveservices.azure.com).")
+    Endpoint: string,
+  }
+)
+@versioned(Versions)
+namespace Language.AnalyzeText;
+
+@doc("Analyze Conversations Service Versions")
+enum Versions {
+  @doc("Version 2022-05-01")
+  @useDependency(Azure.Core.Versions.v1_0_Preview_2)
+  v2022_05_01: "2022-05-01",
+
+  @doc("Version 2023-04-01")
+  @useDependency(Azure.Core.Versions.v1_0_Preview_2)
+  v2023_04_01: "2023-04-01",
+
+  @doc("Version 2023-04-15-preview")
+  @useDependency(Azure.Core.Versions.v1_0_Preview_2)
+  v2023_04_15_preview: "2023-04-15-preview",
+
+  @doc("Version 2023-11-15-preview")
+  @useDependency(Azure.Core.Versions.v1_0_Preview_2)
+  v2023_11_15_preview: "2023-11-15-preview",
+}
+
+alias ServiceTraits = NoRepeatableRequests &
+  NoConditionalRequests &
+  NoClientRequestId;
+
+alias Operations = Azure.Core.ResourceOperations<ServiceTraits>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@post
+@route("/:analyze-text")
+@doc("Request text analysis over a collection of documents.")
+op analyzeText(
+  @doc("Collection of documents to analyze and a single task to execute.")
+  @body
+  body: AnalyzeTextTask,
+
+  ...AnalyzeTextQueryParameters,
+): AnalyzeTextTaskResult | ErrorResponse;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@doc("Get the status of an analysis job. A job can consist of one or more tasks. After all tasks succeed, the job transitions to the succeeded state and results are available for each task.")
+@summary("Get analysis status and results")
+op analyzeTextJobStatus is LanguageLongRunningResourceRead<
+  AnalyzeTextJobState,
+  AnalyzeTextJobQueryParameters
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@doc("Submit a collection of text documents for analysis. Specify one or more unique tasks to be executed as a long-running operation.")
+@route("/analyze-text/jobs")
+@pollingOperation(analyzeTextJobStatus)
+op analyzeTextSubmitJob is LanguageLongRunningOperation<
+  AnalyzeTextJobsInput,
+  AnalyzeTextJobState
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@doc("Cancel a long-running Text Analysis job.")
+@summary("Cancel a long-running Text Analysis job.")
+@pollingOperation(analyzeTextJobStatus)
+@actionSeparator(":")
```

Comment:
Did you find that you need this decorator? Having `@action` should have been enough